scripts/init-project.sh

```diff
@@ -15,7 +15,7 @@ docker-compose exec cli phapp install --no-build
 docker-compose exec cli drush entity-updates -y
 
 # Add demo content.
-docker-compose exec cli drush en contentpool_demo_content -y
+docker-compose exec cli drush en default_content, hal, default_content_extra, file_entity, contentpool_demo_content -y
```

Comment:
Those should probably just be added as dependencies to contentpool_demo_content

Reply:
it is, it's a debug left-over. I'll undo this.